queries/cue/locals.scm

```diff
@@ -19,13 +19,13 @@
 (package_identifier) @local.definition.namespace
 
 (for_clause
-  (identifier) @local.definition.variable
+  (identifier) @local.definition.var
```

Comment:
I don't think this is intended :thinking:

Reply:
Yes it is: https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#locals

(This is exactly why we need to enforce captures at all levels.)

Reply:
To me this seems weird. All other `local.definition.*` are full name, but only `local.definition.var` is shorted :thinking:

Reply:
Agreed. I'll extend it. (But this will be breaking again.)